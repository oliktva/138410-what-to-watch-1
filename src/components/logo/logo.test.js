import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import Logo from './logo';

Enzyme.configure({adapter: new Adapter()});

it(`renders correctly`, () => {
  const logo = shallow(<Logo />);

  expect(toJson(logo)).toMatchSnapshot();
});
