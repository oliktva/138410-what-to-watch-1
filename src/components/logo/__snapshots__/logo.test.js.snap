// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="logo"
>
  <Link
    className="logo__link logo__link--light"
    to="/"
  >
    <span
      className="logo__letter logo__letter--1"
    >
      W
    </span>
    <span
      className="logo__letter logo__letter--2"
    >
      T
    </span>
    <span
      className="logo__letter logo__letter--3"
    >
      W
    </span>
  </Link>
</div>
`;
